lib/loader/serialization/serialize.cpp

```diff
@@ -0,0 +1,258 @@
+#include "ast/module.h"
+#include "loader/loader.h"
+
+namespace WasmEdge {
+namespace Serialize {
+
+std::vector<uint8_t> Serialize::serializeModule(const AST::Module &Mod) {
+  std::vector<uint8_t> serializedModule;
+  std::vector<uint8_t> serializedCustom =
+      serializeCustomSection(Mod.getCustomSections());
+  std::vector<uint8_t> serializedType =
+      serializeTypeSection(Mod.getTypeSection());
+  std::vector<uint8_t> serializedImport =
+      serializeImportSection(Mod.getImportSection());
+  std::vector<uint8_t> serializedFunction =
+      serializeFunctionSection(Mod.getFunctionSection());
+  std::vector<uint8_t> serializedTable =
+      serializeTableSection(Mod.getTableSection());
+  std::vector<uint8_t> serializedmemery =
+      serializeMemorySection(Mod.getMemorySection());
+  std::vector<uint8_t> serializedGlobal =
+      serializeGlobalSection(Mod.getGlobalSection());
+  std::vector<uint8_t> serializedExport =
+      serializeExportSection(Mod.getExportSection());
+  std::vector<uint8_t> serializedStart =
+      serializeStartSection(Mod.getStartSection());
+  std::vector<uint8_t> serializedElement =
+      serializeElementSection(Mod.getElementSection());
+  std::vector<uint8_t> serializedCode =
+      serializeCodeSection(Mod.getCodeSection());
+  std::vector<uint8_t> serializedData =
+      serializeDataSection(Mod.getDataSection());
+  std::vector<uint8_t> serializedDataCount =
+      serializeDataCountSection(Mod.getDataCountSection());
+  return serializedModule;

```

Comment:
It seems like this function always return the empty vector.

Reply:
I removed this temporarily in latest commit because the encoding of expression is remaining. Thanks